api/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest-framework import serializers
+
+from .models import Lyric
+
+class LyricSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lyric
+        fields = ['id', 'title', 'discription', 'content', 'author', 'composer', 'dated_created']
```

Comment:
`LyricSerializer` has been implemented by @RJ-0605 already.
Also, you didn't have to explicitly add `id`, we don't need to accept `date_created` also so you didn't need to add that too. Also, check your spellings. Thanks.
Remove the `serializers.py`